sql/moz-fx-data-shared-prod/accounts_db_derived/accounts_aggregates_v1/query.sql

```diff
@@ -0,0 +1,65 @@
+-- Source tables of this query aren't partitioned and are overwritten daily.
+-- To make this query idempotent we need to make sure that the data for the
+-- current submission_date is not overwritten. If data for the current
```

Comment:
What does `submission_date` mean in this context since it doesn't look like any pings are involved?

Reply:
Yeah, no telemetry is involved here. To be precise, `submission_date` is "date when the database snapshot was made and aggregation was ran". I thought of calling this `day` but went with `submission_date` for consistency with other tables.

Reply:
nit: in that case I'd probably call it something like `last_updated` or something